core/src/main/java/tech/pegasys/eth2signer/core/signing/ArtifactSigner.java

```diff
@@ -12,25 +12,11 @@
  */
 package tech.pegasys.eth2signer.core.signing;
 
-import tech.pegasys.teku.bls.BLS;
-import tech.pegasys.teku.bls.BLSKeyPair;
-import tech.pegasys.teku.bls.BLSSignature;
-
 import org.apache.tuweni.bytes.Bytes;
 
-public class ArtifactSigner {
-
-  private final BLSKeyPair keyPair;
-
-  public ArtifactSigner(final BLSKeyPair keyPair) {
-    this.keyPair = keyPair;
-  }
+public interface ArtifactSigner {
 
-  public String getIdentifier() {
-    return keyPair.getPublicKey().toString();
-  }
+  String getIdentifier();
 
-  public BLSSignature sign(final Bytes message) {
-    return BLS.sign(keyPair.getSecretKey(), message);
-  }
+  ArtifactSignature sign(final Bytes message);
```

Comment:
nit: message or "data"

Reply:
data to be consistent with naming elsewhere in eth2signer